engine/src/main/resources/assets/i18n/menu_en.lang

```diff
@@ -260,7 +260,13 @@
     "telemetry-launch-popup-text": "Telemetry system will send metrics and errors to the server. We do our best to avoid sending anything identifiable from your PC, e.g. telemetry includes your OS name, Java version but it doesn't include the MAC address, the home path, etc. You can review in Metric Menu and let us know if we missed something. Telemetry is super useful to an all volunteer project like ours and we'd really appreciate it!",
     "telemetry-launch-popup-title": "Telemetry In Terasology",
     "telemetry-menu": "Metrics Menu",
-    "telemetry-system-context": "System context",
+    "telemetry-description": "This screen shows the metrics and their values that will be sent to the server. You can enable/disable it here. Telemetry is super useful to an all volunteer project like ours and we'd really appreciate it!",
```

Comment:
In addition to adding new translated English entries in `menu_en.lang` please also add placeholder entries ` menu.lang` so Weblate and such can figure out there are new keys :-)